mustafa-ali-interview.html

```diff
@@ -31,7 +31,7 @@
     </header>
     <div id="aside" class="pv6 mh6 dn-ns dn-m dib-l">
         <div class="aside">
-            <a href="https://envirodatagov.org/wp-content/uploads/2017/06/Part-1-EPA-Under-Siege.pdf" class="aside-el">
```

Comment:
We need this to be on the main page, I dropped the link in slack: https://envirodatagov.org/wp-content/uploads/2017/09/Part-2-Pursuing-Toxic-Agenda.pdf